src/libs/geotagging.c

```diff
@@ -1579,33 +1579,43 @@ static void _timezone_save(dt_lib_module_t *self)
   d->tz_camera = !name ? g_time_zone_new_utc() : g_time_zone_new(name);
   dt_conf_set_string("plugins/lighttable/geotagging/tz", name ? name : "UTC");
   gtk_entry_set_text(GTK_ENTRY(d->timezone), name ? name : "UTC");
+  gtk_label_set_text(GTK_LABEL (d->timezone_changed), "");
+
+  gtk_window_set_focus(GTK_WINDOW(dt_ui_main_window(darktable.gui->ui)), NULL);
+#ifdef HAVE_MAP
+  if(d->map.view)
+    _refresh_track_list(self);
+#endif
 }
 
 static gboolean _timezone_key_pressed(GtkWidget *entry, GdkEventKey *event, dt_lib_module_t *self)
 {
+  dt_lib_geotagging_t *d = (dt_lib_geotagging_t *)self->data;
+
   switch(event->keyval)
   {
     case GDK_KEY_Return:
     case GDK_KEY_KP_Enter:
-      _timezone_save(self);
-      gtk_window_set_focus(GTK_WINDOW(dt_ui_main_window(darktable.gui->ui)), NULL);
-#ifdef HAVE_MAP
-      dt_lib_geotagging_t *d = (dt_lib_geotagging_t *)self->data;
-      if(d->map.view)
-        _refresh_track_list(self);
-#endif
+    case GDK_KEY_Tab:
+      _timezone_save(self, d);
       return TRUE;
     case GDK_KEY_Escape:
       gtk_window_set_focus(GTK_WINDOW(dt_ui_main_window(darktable.gui->ui)), NULL);
       return TRUE;
-    case GDK_KEY_Tab:
-      return TRUE;
     default:
+      gtk_label_set_text(GTK_LABEL (d->timezone_changed), "* ");
```

Comment:
Why the extra space ?
Alignment should be treated in CSS.

Reply:
I tried css, but then there is also a space if no * is shown, so I kept the space